src/main/scala/org/fortysevendeg/sparksftp/common/HiveUserData.scala

```diff
@@ -12,16 +12,14 @@ import SparkUtils._
 object HiveUserData {
 
   /** Sample operations to perform on the user data
-   *
-   *
    */
   def persistUserData(sparkSession: SparkSession, users: DataFrame, salaries: DataFrame) = {
```

Comment:
What's the return type? Even if it's `Unit`, we should put it in the signature

Reply:
Changed in fddb40cc3d1f7d7bae328f4785daba0c06ebf0de